compiler/optimizer/OptimizationPolicy.hpp

```diff
@@ -25,11 +25,21 @@
 namespace TR
 {
 
-class OptimizationPolicy: public TR::Allocatable<OptimizationPolicy, TR::Allocator>
+class OptimizationPolicy
 	{
 	public:
 
-	OptimizationPolicy(TR::Compilation *comp) : _comp(comp) {}
+	static void *operator new(size_t size, TR::Allocator a)
+      { return a.allocate(size); }
```

Comment:
Please fix the indentation in this file to make it consistent with the surrounding code.